app.js

```diff
@@ -26,7 +106,28 @@ if(responseOne){
   alert(promtTwo);
 }
 
-var quizResponseOne = prompt('Please answer with Y or N.  Does Isak have a dog?');
+for(var i = 0; i < 5; i++){
```

Comment:
Where does the number 5 come from? Can you think of a clearer way to do, maybe related to an array?